Backend/app/models/supporter.model.js

```diff
@@ -1,37 +1,53 @@
-const sql = require('./db.js');
-const logger = require('../logger');
+const sql = require("./db.js");
+const logger = require("../logger");
```

Comment:
It seems like double quotes are being prioritized by the editor, can you try installing an eslint plugin to automatically catch styling issues?